web-registry/src/lib/transform.ts

```diff
@@ -38,24 +38,6 @@ export function buildIssuanceModalData(
         const projectDeveloper = getParty(project.partyByDeveloperId);
         if (projectDeveloper) issuees.push(projectDeveloper);
       }
-      if (
```

Comment:
let's delete this file too and `useIssuanceModal`

Reply:
i think there's still some stuff we need in the `transform.ts` file, i.e. `getDisplayParty`. but i was able to remove `buildIssuanceModalData` from there since we don't need that. and `useIssuanceModal` is gone now too.

Reply:
maybe we could put `getDisplayParty` in some ProjectDetails.utils.ts (/web-registry/src/components/templates/ProjectDetails/) and rm transform.ts? it only seems to be used there